src/test/signalk/SKNMEA2000ParserTest.cpp

```diff
@@ -111,6 +111,16 @@ TEST_CASE("SKNMEA2000Parser: Basic tests") {
     CHECK( update.getSteeringRudderAngle() == Approx(SKDegToRad(-3.6)).epsilon(0.0001) );
   }
 
+  SECTION("127275: NavigationAttitude Heel, Pitch, Yaw") {
+    // SID, yaw, pitch, roll in radians
+    SetN2kAttitude(msg, 0, SKDegToRad(179.9), SKDegToRad(-2.3), SKDegToRad(3.2));
+    const SKUpdate &update = p.parse(SKSourceInputNMEA2000, msg, SKTime(0));
+    CHECK( update.getSize() == 1);
+    CHECK( update.getNavigationAttitude().yaw   == Approx(SKDegToRad(179.9)).epsilon(0.0001) );
+    CHECK( update.getNavigationAttitude().pitch == Approx(SKDegToRad(-2.3)).epsilon(0.0001) );
+    CHECK( update.getNavigationAttitude().roll  == Approx(SKDegToRad(3.2)).epsilon(0.0001) );
+  }
```

Comment:
We should had a check with NAN as the Yaw value here. Just to confirm it gives what we want.